src/shell/output.c

```diff
@@ -409,15 +409,17 @@ static int shell_output_file (struct shell_output *out)
 }
```

Comment:
in commit message for 8e13b6ee8e057b2c0a879672e5d357f5867ce76b s/feild/field/ and s/commont/common/